sdks/python/apache_beam/runners/interactive/interactive_environment.py

```diff
@@ -79,11 +79,16 @@ def __init__(self, cache_manager=None):
     self._watching_set = set()
     # Holds variables list of (Dict[str, object]).
     self._watching_dict_list = []
-    # Holds results of pipeline runs as Dict[Pipeline, PipelineResult].
+    # Holds results of main jobs as Dict[Pipeline, PipelineResult].
     # Each key is a pipeline instance defined by the end user. The
     # InteractiveRunner is responsible for populating this dictionary
     # implicitly.
     self._pipeline_results = {}
```

Comment:
Should we change this name to _main_pipeline_results?

Reply:
Changed the name